code/modules/admin/topic.dm

```diff
@@ -958,6 +958,7 @@
 		for(var/datum/job/job as anything in SSjob.joinable_occupations)
 			if(job.title == Add)
 				job.total_positions += 1
+				log_job_debug("[key_name(usr?.client)] added a slot to [job.title]")
```

Comment:
Change all of these to just `key_name(usr)`, it'll automatically grab the client if necessary.